onnx/backend/test/cmd_tools.py

```diff
@@ -80,6 +86,8 @@ def parse_args():  # type: () -> argparse.Namespace
     subparser = subparsers.add_parser('generate-data', help='convert testcases to test data')
     subparser.add_argument('-o', '--output', default=DATA_DIR,
                            help='output directory (default: %(default)s)')
+    subparser.add_argument('-t', '--op_type', default=None,
+                           help='specify the node (lowercase) to exclude other testcases -> generates test data only for the specified op_type')
```

Comment:
reword to something like -> op_type for test case generation. (generates test data for the specified op_type only.)

Reply:
why lowercase? I dont think operator names are lower case Add is valid op_type... for simplicity you can convert both the provided op_type and node.op_type to lower case before comparison... or maybe python has a string comparison api which ignores case

Reply:
Good catch. I use lowercase for these two strings to achieve case insensitivity.

Reply:
Why do we want case-insensitivity? ONNX does not specify operator names are case-insensitive, does it? In which case, it seems better to not change the case at all, and expect the user to specify same case as ONNX.

Reply:
I was afraid that users will use wrong case to update test data and fail, but they should be aware of this while uploading related test data. I agree with you that we should keep it align with case-sensitive ONNX. Just removed lower() to make it case sensitive.